app/api/config/urls.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from django.contrib import admin
+from django.urls import include, path
+
+urlpatterns = [
+    (
+        path("", include("proxy.urls"))
+        if os.environ.get("ENABLE_PROXY", "False").lower() == "true"
+        else None
+    ),
+    path("api/", include("api.urls")),
+    path("", include("mapping.urls")),
```

Comment:
I'm a little sceptical of two paths at `""`. What's happening here?

Reply:
So the first one is the proxy - it's allowing us to override any urls if it's enabled, effectively instead of routing to the Django views, we can forward on to the Next app instead. It's a separate app to make this clear, rather than bury it in `mapping/urls.py`
The second one is the default, that's including the Django views.